src/main/javascript/view/ParameterView.js

```diff
@@ -7,7 +7,7 @@ SwaggerUi.Views.ParameterView = Backbone.View.extend({
 
   initialize: function(){
     Handlebars.registerHelper('isArray', function(param, opts) {
-      if (param.type.toLowerCase() === 'array' || param.allowMultiple) {
+      if (param.type && param.type.toLowerCase() === 'array' || param.allowMultiple) {
```

Comment:
So my use case is actually valid.  Try this spec,

https://s3.amazonaws.com/cti-api-docs/noma-lights/master/swagger.json

By http://editor.swagger.io/ standards it is valid document.  But the swagger-ui chokes on the line in question.

You can see a similar guard introduced here:

https://github.com/swagger-api/swagger-ui/blob/52f3b6d50ec727a8f4d8ddd06388a7edb5d4f97e/src/main/javascript/view/OperationView.js#L816